code/include/env/misa.h

```diff
@@ -1,12 +1,42 @@
 #ifndef RISCV_MISA_H
```

Comment:
I see that you didn't run lint/make/make spike. Fix your PR after running those. Also use better names for you PR, `Updated` is a terrible name.